Project.toml

```diff
@@ -1,14 +1,15 @@
 name = "MLJText"
 uuid = "5e27fcf9-6bac-46ba-8580-b5712f3d6387"
 authors = ["Chris Alexander <uvapazzo@gmail.com>, Anthony D. Blaom <anthony.blaom@gmail.com>"]
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
This version increment indicates a breaking change, according to Julia's take on semantic versioning for 0.?.?
Is somebreaking in the PR?

Reply:
good call - flipped it to 0.1.1.